### DISCUSSION TITLE
Include specific operating system tips?

### DISCUSSION THREAD

**Discussion by bceda** [security-mountain/security-mountain.github.io#3] [Ideas]
Hi,
We should probably clarify for the project's scope whether we will be including specific tips directed towards certain Operating Systems E.g. Windows, iOS, Android.

For example the tip relating to checking Play Protect is enabled, to keep harmful apps of your Android device.

What do you think?

**Comment by ElliottS200** [comment]
I think this is a great idea and should definitely be implemented within tips that differ between OS'

**Reply by bceda** [reply]
How do you think this should be implemented? Should we add in version 2.0.0, a way to ask the user which operating systems/devices they use and then filter out tips that aren't relevant to them. For now we can just expect the user to skip over the ones not relevant.

**Reply by apollodelphi702** [reply]
I second this, for version 2.0 as well as the option to select whether it's personal or business tips the user is looking for, we can add an additional option asking for their OS or devices.
